### PR TITLE
Draft: Added swagger 

### PR DESCRIPTION
**Pull request by koen253janssen**
Added swagger with a helloWorldController for testing, tsoa npm library auto-generates OpenAPI/Swagger spec. I tried changing the defaultController to use tsoa for auto-generating swagger spec. However, doing this would require too much time as all controllers need to be changed to accommodate this.

Swagger didn't work as the helmet library caused SSL issues. This is not resolved by disabling helmet when the app is in development. The development variable for this can be changed in the .env file.

### CODE REVIEW

File: .env
Comment (BartvanEijkelenburg) [inline .env:3]: Please update the .gitignore this file shouldn't be here

File: src/routes.ts
Comment (BartvanEijkelenburg) [inline src/routes.ts:9]: This file aswell

**Review by BartvanEijkelenburg — CHANGES_REQUESTED** [review]

File: .env
Comment (koen253janssen) [reply .env]: Removed

File: src/routes.ts
Comment (koen253janssen) [reply src/routes.ts]: File is removed

**Review by BartvanEijkelenburg — CHANGES_REQUESTED** [review]
The build still fails, I cannot 'approve' this PR if the build is failing, this will destroy the CI/CD, and besides that, I will not approve The PR since this "feature" is not finished, it is only the first setup. Glad that it works but it's not there yet. If you want to discuss this, let me know.

**Pull request closed without merging**
